openrtb/video.go

```diff
@@ -1,30 +1,42 @@
 package openrtb
 
-import "errors"
+import (
+	"errors"
+
+	"github.com/Vungle/vungo/internal/util"
+)
 
 // Video types annotates the parent impression as an video impression.
-// The "protocol", "sequence", "battr", "maxextended", "companionad",
-// "companiontype", and "ext" keys are unused and have been omitted.
-// See OpenRTB 2.3.1 Sec 3.2.4.
+// See OpenRTB 2.5 Sec 3.2.7.
 //go:generate easyjson $GOFILE
 //easyjson:json
 type Video struct {
-	MIMETypes       []string         `json:"mimes"`
-	MinDuration     *int             `json:"minduration,omitempty"`
-	MaxDuration     *int             `json:"maxduration,omitempty"`
-	Protocols       []VideoProtocol  `json:"protocols,omitempty"`
-	Width           int              `json:"w,omitempty"`
-	Height          int              `json:"h,omitempty"`
-	StartDelay      *int             `json:"startdelay,omitempty"`
-	Linearity       Linearity        `json:"linearity,omitempty"`
-	MinBitRate      int              `json:"minbitrate,omitempty"`
-	MaxBitRate      int              `json:"maxbitrate,omitempty"`
-	IsBoxingAllowed NumericBool      `json:"boxingallowed,omitempty"`
-	PlaybackMethods []PlaybackMethod `json:"playbackmethod,omitempty"`
-	DeliveryMethods []DeliveryMethod `json:"delivery,omitempty"`
-	Position        AdPosition       `json:"pos,omitempty"`
-	APIFrameworks   []APIFramework   `json:"api,omitempty"`
-	Extension       interface{}      `json:"ext,omitempty"`
+	MIMETypes                 []string              `json:"mimes"`
+	MinDuration               *int                  `json:"minduration,omitempty"`
```

Comment:
Please using the same format as the comments

Reply:
Done